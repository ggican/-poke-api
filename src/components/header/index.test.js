import React from "react";
import Header from "./index";

let wrapper;
let wrap;

beforeEach(() => {
    wrapper = shallow(
        <MemoryRouter initialEntries={["/random"]}>
            <Header data-test="header-test-wrapper" />
        </MemoryRouter>,
    );
    wrap = mount(
        <MemoryRouter initialEntries={["/random"]}>
            <Header data-test="header-test-wrapper" />
        </MemoryRouter>,
    );
});

describe("Header Component", () => {
    it("renders and create snapshot <Header />", () => {
        expect(wrapper).toMatchSnapshot();
    });

    it("renders", () => {
        expect(wrapper.exists()).toBe(true);
    });
});
