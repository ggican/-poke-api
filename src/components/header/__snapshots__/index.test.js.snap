// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders and create snapshot <Header /> 1`] = `
@media (max-width:450px) {

}

<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "canGo": [Function],
      "createHref": [Function],
      "entries": Array [
        Object {
          "hash": "",
          "key": "eltx9x",
          "pathname": "/random",
          "search": "",
          "state": undefined,
        },
      ],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "index": 0,
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "eltx9x",
        "pathname": "/random",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Header
    data-test="header-test-wrapper"
  />
</Router>
`;
